Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/reflect/LuceneField.java

```diff
@@ -9,7 +9,17 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface LuceneField {
 
-	String lucenefieldName();
-	boolean isTokenized();
+    String lucenefieldName();
+    boolean isTokenized();
     boolean isMultiple();
+
+    /**
+     * This will be set to true if we need to index the field twice (first
+     * tokenized, second non tokenized) to support short word search.
+     * 
+     * @return
+     */
+    boolean isShortWordSearchAllowed() default false;
```

Comment:
this is a bad name. It's not about allowing short words, its about storing and searching for very short words.